docs/source/models/performance.rst

```diff
@@ -16,23 +16,27 @@ You can enable the feature by specifying
     # NOTE: 512 is the default max_num_batched_tokens for chunked prefill.
     # llm = LLM(model="meta-llama/Llama-2-7b-hf", enable_chunked_prefill=True, max_num_batched_tokens=512)
 
-By default, vLLM scheduler prioritizes prefills and doesn't batch prefill and decode to the same batch. This policy optimizes the TTFT (time to thefirst token), but incurs slower ITL (inter token latency) and inefficient GPU utilization.
+By default, vLLM scheduler prioritizes prefills and doesn't batch prefill and decode to the same batch.
+This policy optimizes the TTFT (time to the first token), but incurs slower ITL (inter token latency) and inefficient GPU utilization.
 
-Once chunked prefill is enabled, the policy is changed to
-
-- prioritize decode requests. It batches all pending decode requests to the batch before scheduling any prefill.
-- When there are available token_budget (`max_num_batched_tokens`), it schedules pending prefills. If a last pending prefill request cannot fit into `max_num_batched_tokens`, it chunks it.
+Once chunked prefill is enabled, the policy is changed to prioritize decode requests.
+It batches all pending decode requests to the batch before scheduling any prefill.
+When there are available token_budget (``max_num_batched_tokens``), it schedules pending prefills.
+If a last pending prefill request cannot fit into ``max_num_batched_tokens``, it chunks it.
 
 This policy has two benefits.
 
-- It improves ITL (inter token latency) and generation decode because decode requests are prioritized.
+- It improves ITL and generation decode because decode requests are prioritized.
 - It helps achieve better GPU utilization by locating compute-bound (prefill) and memory-bound (decode) requests to the same batch.
 
-You can tune the performance by changing `max_num_batched_tokens`.
+You can tune the performance by changing ``max_num_batched_tokens``.
 By default, it is set to 512, which has the best ITL on A100 in the initial benchmark.
 Smaller batch size achieves better ITL because there are fewer prefills interrupting decodes.
 Higher batch size achieves better TTFT as you can put more prefill to the batch.
```

Comment:
It seems that we're using `batch size` and `max_num_batched_tokens` interchangeably here, but users might confuse it with `max_num_seqs`. WDYT?

Also please consider making these two as bullet points as well - IMO it will make the documentation a bit easier to read.